src/app/containers/Caption/index.jsx

```diff
@@ -14,36 +14,43 @@ const renderText = textBlocks => (
   <Blocks blocks={textBlocks} componentsToRender={componentsToRender} />
 );
 
-const renderCaption = (block, imageCaptionOffscreenText) => (
+const renderCaption = (block, captionOffscreenText) => (
   <Caption>
-    {imageCaptionOffscreenText ? (
-      <VisuallyHiddenText>{imageCaptionOffscreenText}</VisuallyHiddenText>
+    {captionOffscreenText ? (
+      <VisuallyHiddenText>{captionOffscreenText}</VisuallyHiddenText>
     ) : null}
     {renderText(block)}
   </Caption>
 );
 
-const renderMultipleCaptions = (blocks, imageCaptionOffscreenText) =>
+const renderMultipleCaptions = (blocks, captionOffscreenText) =>
   blocks.map(block =>
-    renderCaption(
-      deepGet(['model', 'blocks'], block),
-      imageCaptionOffscreenText,
-    ),
+    renderCaption(deepGet(['model', 'blocks'], block), captionOffscreenText),
   );
 
-const CaptionContainer = ({ block }) => (
+const CaptionContainer = ({ block, video }) => (
   <ServiceContext.Consumer>
-    {({ imageCaptionOffscreenText }) =>
-      renderMultipleCaptions(
-        deepGet(['model', 'blocks', 0, 'model', 'blocks'], block),
-        imageCaptionOffscreenText,
-      )
-    }
+    {video
```

Comment:
You can reduce the duplication in this block by setting the value of `captionOffscreenText` outside of this. 
Later on, we'll have Audio as well as Video, so we need this to scale, so instead of a boolean value for `video`, you can have value of `type` and switch between them. 

Also, it'd be good to use the React Hook `useContext` instead of the standard `ServiceContext.Consumer` usage we have here.  For example: 
`const { imageCaptionOffscreenText, videoCaptionOffscreenText } = useContext(ServiceContext);`

Reply:
I just checked with Natalie and Ngan-Thi - we should use `Video caption, ` for video and `Audio caption, ` if it's an audio block.


In the Video Container, you can distinguish between Video and Audio blocks by looking at the `kind` field in the `aresMedia` block - if it is `radioProgramme` or `radioClip`, it is Audio. If `programme` or `clip` it is video.